### DISCUSSION TITLE
A beta package - directus@next

### DISCUSSION THREAD

**Discussion by rashidpathiyil** [directus/directus#11555] [Feature Requests]
It will be really useful to have a beta package of Directus, which will available latest commits. 

By using directus@next, we can try new features earlier and report bugs on #beta_chat discord channel.

I think it will improve development speed and also beta users will get fixes soon to continue their development without waiting for a stable release.

**Comment by abdonrd** [comment]
I like this idea!

Or maybe there is a way to use Directus directly by referencing a repository commit?

`npm install directus/directus#<commit>`

**Comment by rijkvanzanten** [comment]
Love it! We'll have to do some more thinking through whether that's something we can/should do every commit, or rather do it on a schedule (like nightly). Our buildchain currently takes quite a while to release (as we're building container images for different architectures etc), so we'll have to think through whether or not we should support docker images in this as well 🤔

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
